### PR TITLE
Compile dsrc under macox

### PR DESCRIPTION
**Pull request by cokelaer**
* [x] I have read the [guidelines for bioconda recipes](https://bioconda.github.io/guidelines.html).
* [ ] This PR adds a new recipe.
* [x] AFAIK, this recipe **is directly relevant to the biological sciences** (otherwise, please submit to the more general purpose [conda-forge channel](https://conda-forge.org/docs/)).
* [x] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

### CODE REVIEW

File: recipes/dsrc/meta.yaml [hunk 1]
```diff
@@ -1,33 +1,37 @@
+{% set name = "dsrc" %}
+{% set version = "2014.12.17" %}
```
Comment (druvus) [inline recipes/dsrc/meta.yaml]: why is version set to a date instead of real version?

Comment (cokelaer) [reply recipes/dsrc/meta.yaml]: I don't remember what was the rationale for that decision. I guess we can switch to 2.0.2 but I'm wondering what would be the effect if we switch to 2.0.2. I mean how the version 2014.12.17 would compare to 2.0.2 Unfortunately 2.0.2 is smaller... but the build number may help if we increase it.

**Comment by cokelaer** [comment]
the macosx compilation fails due to lib boost that cannot be found. I will need to figure out what is missing or wrong in the build.sh file.

**Comment by cokelaer** [comment]
won't fix the macosx version for now. Closing the pull request and will be done later re-using the different proposal to be found in this pull request.

**Pull request closed without merging**